### DISCUSSION TITLE
Unknown Response | Payment source required to redeem gift

### DISCUSSION THREAD

**Discussion by Devabdellah** [marioparaschiv/nitro-sniper#140] [General]
I have a PayPal payment method linked with my Discord, I'm getting this error on every code

**Comment by xavieerx** [comment]
It's due to the epic games nitro event, in due time those codes will go away soon but for now ignore them and if the issue still persists try and add a payment method if you haven't already

**Reply by Devabdellah** [reply]
/(discord.gift|discord.com|discordapp.com\/gifts)\/\w{16,25}/gim
this is the Regex , why would it capture and redeem these "https://discord.com/billing/promotions/RNgGBcA5GqsXZ8jP5SbZFMXp" ?
